source/agg-pipeline-builder/count-pipeline-results.txt

```diff
@@ -0,0 +1,60 @@
+.. _refresh-document-counts:
+.. _count-pipeline-results:
+
+================================
+Count Pipeline Results Documents
+================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can view the number of documents outputted by your pipeline with the 
+:guilabel:`count results` button.
+
+About this Task 
+---------------
+
+When you delete or add a document, the :guilabel:`count results` value on the 
```

Comment:
[suggestion]

Considering rephrasing this as an imperative. Spitballing:

```
When you delete or add a document, you must manually refresh the 
:guilabel:`count results` value on the :guilabel:`Aggregations` tab to 
reflect the new document count.
```

Reply:
Updated!